sys/mips/mips/vm_machdep.c

```diff
@@ -72,6 +72,7 @@ __FBSDID("$FreeBSD$");
 
 #ifdef CPU_CHERI
```

Comment:
`__has_feature(capabilities)`?

Reply:
Hmm, I had probably just not changed that out of trying to minimize the diff, but I can change this one.